docs/extensions/controllerregistration.md

```diff
@@ -8,7 +8,7 @@ Similar to how `CloudProfile` or `Seed` resources get into the system, the Garde
 The specification mainly describes which of Gardener's extension CRDs are managed, for example:
 
 ```yaml
-apiVersion: core.gardener.cloud/v1alpha1
```

Comment:
You could also update `docs/extensions/backupbucket.md` and `docs/extensions/backupentry.md`.

Reply:
What exactly shall I update there?